mecademic_description/urdf/meca_500_r3.ros2_control.xacro

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="meca_500_r3_ros2_control" params="
+    name
+    prefix
+    robot_ip
+    control_port
+    monitoring_port
+    velocity_timeout
+    velocity_limit_percent
+    acceleration_limit_percent
+    use_fake_hardware:=false
+    fake_sensor_commands:=false
+    initial_positions:=${dict(axis_1_joint=0.0,axis_2_joint=0.0,axis_3_joint=0.0,axis_4_joint=0.0,axis_5_joint=0.0,axis_6_joint=0.0)}">
```

Comment:
This is a very interesting way of passing the arguments with a dict, I have never seen this. Do you know how you would have to do in a launch file to generate this dict? would it just be a python dict?

Anyway, do we even need this, because you are not exposing those initial positions as argument in the urdf.xacro file?